.github/actions/setup-jdks/action.yml

```diff
@@ -20,7 +20,8 @@ runs:
     - name: 'Set up JDK 8'
       uses: actions/setup-java@v4
       with:
-        distribution: 'temurin'
+        # Temurin JDK 8 for macos on ARM is not available: https://github.com/adoptium/adoptium/issues/96
+        distribution: ${{ runner.os == 'macOS' && 'zulu' || 'temurin' }}
```

Comment:
I'd even say
```suggestion
        distribution: ${{ (runner.os == 'macOS' && runner.arch == 'ARM64') && 'zulu' || 'temurin' }}
```
:-)

Reply:
Too late. Please apply it in one of your further related PRs.

Reply:
Yes, only the latest is arm